resources/qml/dialogs/ImagePackEditorDialog.qml

```diff
@@ -219,6 +221,7 @@ ApplicationWindow {
                         Layout.columnSpan: 2
                         url: imagePack.data(imagePack.index(currentImageIndex, 0), SingleImagePackModel.Url).replace("mxc://", "image://MxcImage/")
                         displayName: imagePack.data(imagePack.index(currentImageIndex, 0), SingleImagePackModel.ShortCode)
+                        userid: displayName
```

Comment:
I don't think a jdenticon for a sticker akes sense. tbh, we should be using an Image here, not an Avatar...

Reply:
Well, it currently loads a letter as the placeholder, so I thought I'd add jdenticon... want me to go ahead and change it to an Image?

Reply:
Wait, I thought this place was for the actual stickers... I think I just confused the lines :D

Reply:
Is there any sort of statekey I can use here as well, or is what I have fine?

Reply:
You can use the shortcode, which probably makes more sense